modules/ROOT/pages/performance-troubleshooting.adoc

```diff
@@ -108,9 +108,21 @@ that take volume in the database and that slows some queries down.
 
 == Connectors
 
-Any connectors longer than 10 seconds will produce a warning log. See xref:performance-tuning.adoc]#connector_service[connector service] for more information.
-
-In addition, performance of connectors can be measured using the xref:performance-tuning.adoc]#time_tracker[connector time tracker].
+If a connector execution duration exceeds the configured timeout (300 seconds by default), an exception will be raised in the log: 
+[source,log]
+----
+Caused by: org.bonitasoft.engine.connector.exception.SConnectorException: The connector timed out
+---- 
+[NOTE]
+====
+for subscription editions only.
```

Comment:
⚠️ This is not the wording we use in other page.

![image](https://user-images.githubusercontent.com/27200110/135838915-9616aa0f-3554-44c9-aa04-c779e8cc531f.png)

or "In Teamwork, Efficiency, Performance and Enterprise editions"